cmd/seeder/main.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/gorilla/mux"
+	_ "github.com/mattn/go-sqlite3"
+)
+
+var products []product
+
+type product struct {
+	Id         int    `json:"id"`
+	Name       string `json:"name"`
+	Price      int    `json:"price"`
+	Descrition string `json:"description"`
+}
+
+func main() {
+
+	var dbProducts string
+	var jsonProducts string
```

Comment:
Use better names for these. Given their intent is to be file paths, maybe name them accordingly.